packages/css/src/common/size.scss

```diff
@@ -3,11 +3,12 @@
  * Copyright Gemeente Amsterdam
```

Comment:
Copied from a PR comment: 

> By the way, I think we should mark (non-trivial) documentation updates as features from now on. Documentation on how to use things is an important part of our design system and they deserve a place in the change log.

That creates a kind of weird situation though. The documentation lives in Storybook, which is a package we don't publish. Marking it as a feature without changing something in the packages we do publish doesn't result in a new version being released. And vice versa, if we do release a new version of the packages we publish, it's because there's some other change in that package, not because of changes to the docs.

Reply:
Fair point. Still I think it’s important to point people to new or updated guidelines. Let’s think of something else.

Reply:
Robbert once mentioned creating a package for docs, that could be an interesting option. We'd have to decide on the format though.

Reply:
https://gemeente-amsterdam.atlassian.net/browse/DES-925